stock_picking_group_by_partner_by_carrier_by_date/models/stock_move.py

```diff
@@ -25,10 +32,22 @@ def _assign_picking_group_domain_by_date(self):
         domain = []
         if self._skip_assign_picking_group_domain_by_date():
             return domain
-        date_planned = self.date.replace(hour=0, minute=0, second=0)
-        date_planned_end = date_planned.replace(hour=23, minute=59, second=59)
+        tz = (
```

Comment:
@tuantrantg Is there a test for timezones ?

Reply:
@tuantrantg little ping, any news here ?

Reply:
I'm a little bit busy right now, I will come back soon

Reply:
hi @rousseldenis the test is added, could you please take a look